src/components/ContractDetailsTab.js

```diff
@@ -1,15 +1,26 @@
 import React, { Component, Fragment } from "react";
 import { Tab, Grid, Typography } from "@material-ui/core";
 import { formatMessage, PublishedComponent, FormattedMessage } from "@openimis/fe-core";
-import { RIGHT_POLICYHOLDERCONTRACT_UPDATE, RIGHT_POLICYHOLDERCONTRACT_APPROVE, CONTRACTDETAILS_TAB_VALUE } from "../constants"
+import {
+    RIGHT_POLICYHOLDERCONTRACT_UPDATE,
+    RIGHT_POLICYHOLDERCONTRACT_APPROVE,
+    CONTRACTDETAILS_TAB_VALUE,
```

Comment:
why there is no RIGHT_POLICYHOLDERCONTRACT_SEARCH ?

Reply:
@delcroip `ContractDetailsTab` is displayed on Contract create (as disabled) and update page.

As Contract Details inherit rights/permissions from Contract, this tab is shown if a (normal, NOT Portal) user is permitted to access Contract update page. `RIGHT_POLICYHOLDERCONTRACT_SEARCH` is therefore not needed, as this right/permission is used only on Contract search page.

On the other hand, Contract Details have their own rights/permissions for Portal users (`portalPHCD`). In order to display `ContractDetailsTab` to a Portal user `RIGHT_PORTALPOLICYHOLDERCONTRACT_SEARCH` is checked.

Reply:
thx